front/src/common/redux/set-password/set.password.api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function setPassswordApi (request) {
+    const url = `${process.env.REACT_APP_SET_PASSWORD_ROUT}${request.data.token}`;
+    const headers = { headers: { 'Content-Type': 'application/json' } };
+   
+    return axios
+        .put(url, JSON.stringify(request.data), headers)
```

Comment:
We don't need `JSON.stringify` here, just pass `request.data` here

Reply:
Fixed